pyblish_lite/control.py

```diff
@@ -95,7 +95,7 @@ def on_next():
 
         util.defer(100, on_next)
 
-    def emit_(self, signal, kwargs):
+    def emit_(self, signal, **kwargs):
```

Comment:
Haha, alright :) Thought it was a bit weird you wanted to deviate from the QML structure.